src/components/Modal.js

```diff
@@ -60,8 +61,8 @@ const Modal = React.createClass({
 
   componentDidMount () {
   /*eslint-disable */
-    if (this.props.isOpen !== null) {
-      console.warn('WARNING: The prop "isOpen" is depracated in this version of the component. Please handle Modal opening from its parent.');
+    if (_has(this.props, 'isOpen')) {
```

Comment:
In that case, wouldn't the warning NOT be shown when the prop is present but set as false?  I would think we would want to show the warning all the time if the prop is present.